README.md

```diff
@@ -47,6 +47,8 @@ Although Gradle offers a large degree of flexibility in your project structure,
 
 **General structure.** Follow [Google's guide on Gradle for Android](https://developer.android.com/studio/build/index.html).
 
+**minSdkVersion: 21** We recommend to have a look at the [Android version usage chart](https://developer.android.com/about/dashboards/index.html#Platform) before defining the minimum API required. It is worth to mention that some material design features are only available on Android 5.0 (API level 21) and above. And also, from API 21, the multidex support library is not needed anymore.
```

Comment:
Also add a mention that the statistics given on that chart are global statistics, so when targeting a specific regional/demographic market, you might need to obtain more specific device usage information.